File: problems/3452-sum-of-good-numbers/solution.c
int sumOfGoodNumbers(int* nums, int numsSize, int k) {
    int sum = 0;
    for (int i = 0; i < numsSize; i++) {
        int isGood = 1;
        if (i - k >= 0 && nums[i] <= nums[i - k]) {
            isGood = 0;
        }
        if (i + k < numsSize && nums[i] <= nums[i + k]) {
            isGood = 0;
        }
        if (isGood) {
            sum += nums[i];
        }
    }
    return sum;
}

File: problems/3452-sum-of-good-numbers/solution.java
class Solution {
    public int sumOfGoodNumbers(int[] nums, int k) {
        int sum = 0;
        for (int i = 0; i < nums.length; i++) {
            boolean good = true;
            if (i - k >= 0 && nums[i] <= nums[i - k]) {
                good = false;
            }
            if (i + k < nums.length && nums[i] <= nums[i + k]) {
                good = false;
            }
            if (good) {
                sum += nums[i];
            }
        }
        return sum;
    }
}